nss-day-cohort-60/django-taters-api
# Evn add404 and 200 status codes

# Description of Proposed Changes
added 404 status code to retrieve and 200 to list and retrieve for authors
---

# Steps to Test
1. pull the branch down to local machine
2. test in postman to ensure 200 on list and individual (http://localhost:8000/authors and http://localhost:8000/authors/1)
3. test in postman to ensure 404 on non-existing data (http://localhost:8000/authors/6)
